boxoffice/views/admin_discount.py

```diff
@@ -52,12 +62,38 @@ def jsonify_discount_policies(data_dict):
     )
 @render_with({'text/html': 'index.html', 'application/json': jsonify_discount_policies}, json=True)
 def admin_discount_policies(organization):
+    RESULTS_PER_PAGE = 6
     query = request.args.get('search')
-    if query:
-        discount_policies = DiscountPolicy.query.filter(DiscountPolicy.title.ilike('%{query}%'.format(query=query))).all()
+    try:
+        page = int(request.args.get('page', 1))
+    except ValueError:
+        page = 1
+
+    if request.is_xhr:
+        discount_policies = DiscountPolicy.query.filter(
+            DiscountPolicy.organization == organization
+        ).order_by('created_at desc')
```

Comment:
Perhaps better to set this as the default sort order and use `organization.discount_policies`.